From 18877ff1165bfc3ca57f440bf4d6ef69b6371830 Mon Sep 17 00:00:00 2001
From: Hui Wang <Hui.Wang@windriver.com>
Date: Tue, 16 Aug 2011 17:43:42 +0800
Subject: [PATCH 21/21] mach_omap2: fix a build error

If ti_sitara_am389x build pm.c, it will produce link error, so remove
it from Makefile.

Signed-off-by: Hui Wang <Hui.Wang@windriver.com>
---
 arch/arm/mach-omap2/Makefile |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/Makefile b/arch/arm/mach-omap2/Makefile
index 1d00561..dc9926f 100644
--- a/arch/arm/mach-omap2/Makefile
+++ b/arch/arm/mach-omap2/Makefile
@@ -3,7 +3,11 @@
 #
 
 # Common support
-obj-y := id.o io.o control.o mux.o devices.o serial.o gpmc.o timer-gp.o pm.o
+obj-y := id.o io.o control.o mux.o devices.o serial.o gpmc.o timer-gp.o
+
+ifndef CONFIG_ARCH_TI816X
+obj-y += pm.o
+endif
 
 omap-2-3-common				= irq.o sdrc.o prm2xxx_3xxx.o
 hwmod-common				= omap_hwmod.o \
-- 
1.7.0.2

